src/style/_fonts.scss

```diff
@@ -32,3 +32,48 @@
     src: url('./fonts/robotomono-regular-webfont.woff2') format('woff2'),
     url('./fonts/robotomono-regular-webfont.woff') format('woff')
 }
+
+
+// Fontello Icons
```

Comment:
**Change request**: add a comment that tells other folks how to regenerate this code (including any manual changes that were made to what Fontello generated).

Will come in handy when/if we need to add a new icon to the mix.